dcos_test_utils/dcos_cli.py

```diff
@@ -92,6 +94,45 @@ def clear_cli_dir():
         if os.path.exists(path):
             shutil.rmtree(path)
 
+    def exec(self, cmd: str, stdin=None, check=True) -> subprocess.CompletedProcess:
+        """Execute CLI command and processes result.
+
+        This method expects that process won't block.
+
+        :param cmd: Program and arguments
+        :type cmd: str
+        :param stdin: File to use for stdin
+        :param check: Does it check for raised errors
+        :type stdin: File
```

Comment:
It isn't `File` always as it is `None` by default. I'm not sure about this syntax, but I would usually use `typing.optional[File]` as a type hint and remove this.